exporter/newrelicexporter/newrelic.go

```diff
@@ -330,16 +331,20 @@ func (e *exporter) pushMetricData(ctx context.Context, md pdata.Metrics) error {
 	var errs []error
 
 	ocmds := internaldata.MetricsToOC(md)
-	for _, ocmd := range ocmds {
+	for index, ocmd := range ocmds {
 		var srv string
+
 		if ocmd.Node != nil && ocmd.Node.ServiceInfo != nil {
 			srv = ocmd.Node.ServiceInfo.Name
 		}
 
+		language, _ := md.ResourceMetrics().At(index).Resource().Attributes().Get(instrumentationLanguageKey)
```

Comment:
This seems not ideal, but should be ok for now. In the event that there is no resource or no `telemetry.sdk.language` attribute on the resource this line sets `language` to an empty string.

This will all change when we remove the initial transform to OpenCensus metrics.